src/ng-option-highlight/lib/ng-option-highlight.directive.ts

```diff
@@ -1,60 +1,60 @@
-import { AfterViewInit, Directive, ElementRef, Input, OnChanges, Renderer2 } from '@angular/core';
+import { afterNextRender, computed, effect, inject, input, signal, Directive, ElementRef, Renderer2 } from '@angular/core';
+
+function isDefined(value: any) {
```

Comment:
I am a fan of having Angular logic and anything else separated so utils in some else file but also something for another pr. Also do place REGEX on the root scope because browsers cache/compile the otherwise everytime you call the function